reference/forms/types/date.rst

```diff
@@ -155,6 +155,19 @@ values for the year, month and day fields::
 
 .. include:: /reference/forms/types/options/view_timezone.rst.inc
 
+``calendar``
+~~~~~~~~~~~~
+
+**type**: ``\IntlCalendar`` **default**: ``null``
```

Comment:
```suggestion
**type**: ``\IntlCalendar|null`` **default**: ``null``
```

Reply:
In the majority of cases, we don't add `|null` to nullable options of form types. I could only find 3 places where this is done. Is there a guideline for this?

Reply:
then let's keep it as is